### ISSUE TITLE
Disabling TextSecure messages and opening group causes crash

### ISSUE THREAD

**Issue by mcginty** [WhisperSystems/Signal-Android#3157]
```
        AndroidRuntime  D  Shutting down VM
                        E  FATAL EXCEPTION: main
                        E  Process: org.thoughtcrime.securesms, PID: 6583
                        E  java.lang.AssertionError: Selected type isn't present!
                        E      at org.thoughtcrime.securesms.TransportOptions.getSelectedTransport(TransportOptions.java:66)
                        E      at org.thoughtcrime.securesms.TransportOptions.notifyTransportChangeListeners(TransportOptions.java:123)
                        E      at org.thoughtcrime.securesms.TransportOptions.setTransport(TransportOptions.java:118)
                        E      at org.thoughtcrime.securesms.TransportOptions.setDefaultTransport(TransportOptions.java:49)
                        E      at org.thoughtcrime.securesms.components.SendButton.setDefaultTransport(SendButton.java:77)
                        E      at org.thoughtcrime.securesms.ConversationActivity.initializeSecurity(ConversationActivity.java:656)
                        E      at org.thoughtcrime.securesms.ConversationActivity.onResume(ConversationActivity.java:209)
                        E      at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1241)
                        E      at android.app.Activity.performResume(Activity.java:6019)
                        E      at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2996)
                        E      at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3038)
                        E      at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2412)
                        E      at android.app.ActivityThread.access$800(ActivityThread.java:149)
                        E      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1324)
                        E      at android.os.Handler.dispatchMessage(Handler.java:102)
                        E      at android.os.Looper.loop(Looper.java:211)
                        E      at android.app.ActivityThread.main(ActivityThread.java:5321)
                        E      at java.lang.reflect.Method.invoke(Native Method)
                        E      at java.lang.reflect.Method.invoke(Method.java:372)
                        E      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1016)
                        E      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:811)
               Process  I  Sending signal. PID: 6583 SIG: 9```

**Comment by rhodey** [comment]
```java
private void initializeSecurity() {
    boolean isMediaMessage = !recipients.isSingleRecipient() || attachmentManager.isAttachmentPresent();
    this.isEncryptedConversation = DirectoryHelper.isPushDestination(this, getRecipients());

    sendButton.resetAvailableTransports(isMediaMessage);

    if (!isEncryptedConversation)      sendButton.disableTransport(Type.TEXTSECURE);
    if (recipients.isGroupRecipient()) sendButton.disableTransport(Type.SMS);

    if (isEncryptedConversation) sendButton.setDefaultTransport(Type.TEXTSECURE);
    else                         sendButton.setDefaultTransport(Type.SMS);

    calculateCharactersRemaining();
}
```

`DirectoryHelper.isPushDestination()` returns false because TS messages are disabled, so `isEncryptedConversation == false` and thus `sendButton.setDefaultTransport(Type.SMS)` fails because that transport was disabled by `recipients.isGroupRecipient() == true`

whatever fix is created for this should also take #2161 into account, should be easy to include.

**Comment by dcherian** [comment]
Not reproducible on 3.4.0
